api-reference/10 UI Components/dxValidationSummary/3 Methods/refreshValidationGroup().md

```diff
@@ -3,7 +3,234 @@ id: dxValidationSummary.refreshValidationGroup()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Refreshes the ValidationSummary subscription to a target [ValidationGroup](/Documentation/ApiReference/UI_Components/dxValidationGroup/).
```

Comment:
I'm not sure how you can 'refresh' something 'to' something else

Reply:
Maybe "Resubscribes the ValidationSummary to a target ValidationGroup"? I meant "refreshes the subscription" and "subscription to a ValidationGroup", like the "to" doesn't apply to "refresh". Maybe I need to use different preposition here?

Reply:
Yes, because if it 'resubscribes', that is not the same as refresh